src/test/java/api/requests/ItemsLostRequiringActualCostsTests.java

```diff
@@ -0,0 +1,263 @@
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+
+import org.folio.circulation.support.http.client.Response;
+import org.hamcrest.CoreMatchers;
+import org.joda.time.DateTime;
+import org.junit.Test;
+
+import api.support.builders.CheckOutByBarcodeRequestBuilder;
+import api.support.builders.DeclareItemLostRequestBuilder;
+import api.support.builders.ItemBuilder;
+import api.support.builders.LostItemFeePolicyBuilder;
+import api.support.http.IndividualResource;
+import api.support.http.ItemResource;
+import api.support.http.ResourceClient;
+import api.support.matchers.ItemMatchers;
+import api.support.spring.SpringApiTest;
+import io.vertx.core.json.JsonObject;
+
+public class ItemsLostRequiringActualCostsTests extends SpringApiTest {
+
+  @Test
+  public void shouldIncludeAnyLostItemsRequiringActualCostFees() {
+    final ItemResource closedItem = createClosedItem();
+    final ItemResource openItem = createOpenItem();
+
+    final LostItemFeePolicyBuilder setCostPolicy = createSetCostChargeFeePolicy();
+    final LostItemFeePolicyBuilder setCostAgedLostPolicy = createSetCostAgeToLostAndBilledAfterOneMinutePolicy();
+    final LostItemFeePolicyBuilder actualCostPolicy = createActualCostChargeFeePolicy();
+    final LostItemFeePolicyBuilder actualCostAgedLostPolicy = createActualCostAgeToLostAndBilledAfterOneMinutePolicy();
+
+    final ItemResource declaredLostSetCostItem = createDeclaredLostItem(setCostPolicy);
+    final ItemResource agedToLostSetCostItem = createAgedToLostItem(setCostAgedLostPolicy);
+    final ItemResource declaredLostActualCostItem = createDeclaredLostItem(actualCostPolicy);
+    final ItemResource agedToLostActualCostItem = createAgedToLostItem(actualCostAgedLostPolicy);
+
+    final Collection<JsonObject> items = ResourceClient.forItemsLostRequiringActualCosts().getAll();
+
+    assertThat(items.size(), is(2));
+    validateItemExistsInItems(items, declaredLostActualCostItem);
+    validateItemInItemsIsDeclaredLost(items, declaredLostActualCostItem);
+    validateItemExistsInItems(items, agedToLostActualCostItem);
+    validateItemInItemsIsAgedToLost(items, agedToLostActualCostItem);
+    validateItemNotExistsInItems(items, closedItem);
+    validateItemNotExistsInItems(items, openItem);
+    validateItemNotExistsInItems(items, declaredLostSetCostItem);
+    validateItemNotExistsInItems(items, agedToLostSetCostItem);
+  }
+
+  @Test
+  public void shouldIncludeItemsThatAreDeclaredLostUsingActualCostCharge() {
+    final LostItemFeePolicyBuilder actualCostPolicy = createActualCostChargeFeePolicy();
+    final ItemResource declaredLostActualCostItem = createDeclaredLostItem(actualCostPolicy);
+
+    final Collection<JsonObject> response = ResourceClient.forItemsLostRequiringActualCosts().getAll();
+
+    assertThat(response.size(), is(1));
+    validateItemExistsInItems(response, declaredLostActualCostItem);
+    validateItemInItemsIsDeclaredLost(response, declaredLostActualCostItem);
+  }
```

Comment:
How is this different to the test above it? Is it intended to be a more specific description of the same functionality?

Reply:
The first tests goal is really just to check that there are multiple items returned.
Using items configured differently to catch multiple cases.
I just wanted to make sure more than 1 items/records are returned.

This next test, shouldIncludeItemsThatAreDeclaredLostUsingActualCostCharge(), is specifically testing that a Declared Lost item is returned. (I thought I had another function that also tested exclusively for aged to lost..maybe the test got lost in the rewrite.) Should I remove this second test given that the first effectively proves this test case?

Reply:
I don't think there is a need to remove the more specific test.

I think it might be worth changing the first test to use the simplest examples as possible, if the goal is only to present multiple results and not care how those results came about.